lib/Doctrine/Common/EventManager.php

```diff
@@ -68,7 +70,7 @@ public function dispatchEvent($eventName, EventArgs $eventArgs = null)
      *
      * @return array The event listeners for the specified event, or all event listeners.
      */
-    public function getListeners($event = null)
+    public function getListeners(?string $event = null): array
```

Comment:
shouldn't this read `string $event = null` without the `?` prefix ?

If the parameter has a default value of `null` anyway, is the `?` not redundant ?